src/pages/_error.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Rollbar from 'rollbar';
+
+function Error({ statusCode }) {
+  return (
+    <p>
+      {statusCode
+        ? `An error ${statusCode} occurred on server!`
+        : 'An error occurred on client'}
+    </p>
+  );
+}
+
+Error.getInitialProps = ({ req, res, err }) => {
+  let statusCode;
+  if (res) {
+    statusCode = res.statusCode;
+  } else if (err) {
+    statusCode = err.statusCode;
+  } else {
+    statusCode = 404;
+  }
+
+  // Only require Rollbar and report error if we're on the server & it's not a local 404.
+  if (!process.browser && statusCode !== 404) {
+    const rollbar = new Rollbar({
+      accessToken: process.env.NEXT_PUBLIC_ROLLBAR_SERVER_TOKEN,
```

Comment:
May be worth erroring if `NEXT_PUBLIC_ROLLBAR_SERVER_TOKEN` doesn't exist? (otherwise we may not know that it's not set)

Reply:
Can't do this in the nice way you do it on ysm-backend because then the build will fail as it doesn't have access to the env variables at build time.

Have added some extra checks in that do the job (albeit not as nicely!)